tests/python/contrib/test_hexagon/rpc/conftest.py

```diff
@@ -1,31 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-import pytest
-
-
-def pytest_addoption(parser):
```

Comment:
I want to use serial number to distinguish running it on target (where it's a serial number of an actual device), and on simulator, where it would be a string "simulator".  `ANDROID_SERIAL_NUMBER` has multiple users.

If you have a better mechanism in mind for setting simulator as the execution platform, let me know.  I'd like to be able to run the same `test.py` on either hardware or simulator without manual editing, so some parts of `HexagonLauncher` will need to be modified a bit (to hide the internals a bit more).

Reply:
This doesn't have to happen in this PR, so if it's ready to merge, then go ahead.

Reply:
@kparzysz-quic that's a good point. we can use something like `board` where it specifies the dev board(HDK, etc) or phone type(OPPO, etc). One of the boards could be`hexagon-simulator` for running on the simulator.
`board` is used at HexagonLauncher level and it could also map to the correct tvm target that is passed to compiler. Something similar to microtvm here: https://github.com/apache/tvm/blob/main/python/tvm/target/target.py#L330

Reply:
In microtvm we go from `board` to `model` and `model` specifies the full target for TVM. Here we could have a direct mapping from `board` to tvm target or we can use both `board` and `cpu_version` to build the tvm target. Not sure if need the latter since it seems unnecessarily complicated

Reply:
I'm not really a fan of the "board" approach for Hexagon, I don't think it's a useful abstraction for our situation.  Moreover, the TVM target will be the same for both hardware and simulator, so we need something else to indicate where the code needs to be executed by the launcher.  This is specific to launcher and nothing else.